packages/core/src/components/fields/SchemaField.js

```diff
@@ -72,14 +72,14 @@ function Label(props) {
 }
 
 function LabelInput(props) {
-  const { id, label, onChange } = props;
   return (
-    <input
-      className="form-control"
-      type="text"
-      id={id}
-      onBlur={event => onChange(event.target.value)}
-      defaultValue={label}
+    <SchemaField
+      {...props}
```

Comment:
Nice, great you figured it out!

Since props can have other attributes like the id attribute, which we don't need to pass to SchemaField, could we instead explicitly name / pass down all props instead of doing {...props}?

Reply:
Thank you for your advice! I just updated the lines to explicitly use attributes from props :)